www/routes/docs/index.tsx

```diff
@@ -1,19 +1,10 @@
-import type { Handlers } from "fresh/compat";
-
-export const handler: Handlers<void> = {
-  GET(ctx) {
-    const slug = ctx.params.slug;
```

Comment:
It was unclear to me how this handler could ever have `ctx.params.slug`, since it's not a dynamic route, so I removed.

But if there is another way and I'm mistaken, LMK and we can add this back!